### PR TITLE
feat: add a noop docker container for SD

### PR DESCRIPTION
**Pull request by jithine**
## Context

Add a noop  container for running Screwdriver bulds

## Objective

Provide a bare minimum container for running builds in Screwdriver

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by scr-oath — APPROVED** [review]
LGTM

**Comment by jithine** [comment]
will update README with a follow up PR once docker build is setup

**Review by anilkumarmyla — APPROVED** [review]
Awesome

**Review by vnugopal — APPROVED** [review]

**Comment by DekusDenial** [comment]
My understanding of "noop" is that i will neither expect to run anything within nor use as a base image. This feels more like a bare min/slim than a true noop. Just saying.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Based on https://github.com/screwdriver-cd/launcher/blob/master/Dockerfile
+RUN set -x \
+    # Alpine ships with musl instead of glibc (this fixes the symlink)
+    # Ref https://docs.screwdriver.cd/user-guide/FAQ#how-do-i-fix-build-failed-to-start-error-message
+    && mkdir /lib64 \
+    && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2 \
+    # Need latest ca-certificates or post to https end-points can fail
+    && apk add --no-cache --update ca-certificates \
+    # Needed for sd-setup-screwdriver-sshca-bookend
+    && apk add --no-cache curl openssh \
```
Comment (DekusDenial) [inline Dockerfile:15]: `curl` depends on `ca-certificates` so you can save up one line

**Review by DekusDenial — APPROVED** [review]

**Pull request merged**